lib/companies/pending_changes.ex

```diff
@@ -5,26 +5,33 @@ defmodule Companies.PendingChanges do
 
   import Ecto.Query
 
-  alias Companies.{Repo, Slack}
+  alias Companies.Repo
   alias Companies.Schema.{Company, Job, Industry, PendingChange}
 
   def all do
     query =
       from p in PendingChange,
-        select: p,
-        where: p.approved == false
+        where: is_nil(p.approved)
```

Comment:
You need to add a migration to make them not default to false in db for this to work.